docs/hotplug.md

```diff
@@ -0,0 +1,55 @@
+# Cloud Hypervisor Hot Plug
+
+Currently Cloud Hypervisor only support hot plugging of CPU devices.
+
+## Kernel support
+
+For hotplug on Cloud Hypervisor ACPI GED support is needed. This can either be achieved by turning on `CONFIG_ACPI_REDUCED_HARDWARE_ONLY` 
+or by using this kernel patch: https://git.kernel.org/pub/scm/linux/kernel/git/rafael/linux-pm.git/commit/?id=ac36d37e943635fc072e9d4f47e40a48fbcdb3f0
+
```

Comment:
The patch is upstream now, since 5.5-rc1:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/patch/drivers/acpi/Makefile?id=ac36d37e943635fc072e9d4f47e40a48fbcdb3f0

Reply:
That's good. I hope when 5.5 gets released, we'll have both this patch and virtio-fs, this would simplify the amount of patches that we're carrying on our custom branch.

Reply:
Updated.